### ISSUE TITLE
Update plugin model documentation and testing (Trac #668)

### ISSUE THREAD

**Issue by smk78** [SasView/sasview#799]
Paraphrased from PaulK:

In sasview/src/sas/sasgui/perspectives/fitting/media/plugins.rst :

1) In the section "Test Your New Model" only the first paragraph currently applies to the release version.  The run_one() and compare.main() functions are only implemented in the developer version.

When these functions are implemented in the release version we might want to consider providing menu bar options for "Compare", "Compare Random" and "Show Help" in addition to the complicated command line access described?

2) Pylint is not currently shipped with the release version so the sections "Clean Lint" or "Check the Docs" are tagged as "Developer Version Only". Obviously this will need updating if the situation changes.
  

Migrated from http://trac.sasview.org/ticket/668
```json
{
    "status": "closed",
    "changetime": "2019-03-26T10:56:06",
    "_ts": "2019-03-26 10:56:06.534352+00:00",
    "description": "Paraphrased from PaulK:\n\nIn sasview/src/sas/sasgui/perspectives/fitting/media/plugins.rst :\n\n1) In the section \"Test Your New Model\" only the first paragraph currently applies to the release version.  The run_one() and compare.main() functions are only implemented in the developer version.\n\nWhen these functions are implemented in the release version we might want to consider providing menu bar options for \"Compare\", \"Compare Random\" and \"Show Help\" in addition to the complicated command line access described?\n\n2) Pylint is not currently shipped with the release version so the sections \"Clean Lint\" or \"Check the Docs\" are tagged as \"Developer Version Only\". Obviously this will need updating if the situation changes.\n  ",
    "reporter": "smk78",
    "cc": "",
    "resolution": "fixed",
    "workpackage": "SasView Documentation",
    "time": "2016-09-27T14:44:08",
    "component": "SasView",
    "summary": "Update plugin model documentation and testing",
    "priority": "major",
    "keywords": "",
    "milestone": "SasView Next Release +1",
    "owner": "smk78",
    "type": "defect"
}
```

**Comment by butlerpd** [comment]
Trac update at `2019/03/23 14:57:09`: **butler** commented:

Has this ticket become obsolete?

**Comment by smk78** [comment]
Trac update at `2019/03/26 10:46:04`: 
* **smk78** commented:

In changeset c94ab04e18bfbb17455ae74d31a271d9cdb0f3b4:

```text
#!CommitTicketReference repository="sasmodels" revision="c94ab04e18bfbb17455ae74d31a271d9cdb0f3b4"
Clarified Testing section of plugin.rst. Closes #799
```


* **smk78** changed owner from "" to "smk78"
* **smk78** changed resolution from "" to "fixed"
* **smk78** changed status from "new" to "closed"

**Comment by smk78** [comment]
Trac update at `2019/03/26 10:56:06`: **smk78** commented:

To address 1) a clarification has been added to the Testing section of plugin.rst. The subsequent suggestion for GUI buttons are now the subject of #1282.

The situation regarding 2) has not changed and the relevant sections of plugin.rst are ok as is.

This ticket is now obsolete so can be closed.
